Sources/SwiftCBOR/CBOREncoder.swift

```diff
@@ -1,336 +1,914 @@
-#if canImport(Foundation)
+//
+//  CBOREncoder.swift
+//  Sunday
+//
+//  Created by Kevin Wooten on 7/11/18.
+//  Copyright © 2018 Outfox, Inc. All rights reserved.
+//
+// This file was adapted from `JSONEncoder.swift` in the
+// `swift-corelibs-foundation` project which carries the following
+// copyright.
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
```

Comment:
SwiftCBOR is currently under the UNLICENSE, this would mean adding apache2 (carrying this Apple copyright etc)